### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by KoditkarVedant**
We are adding reference of Console Application Project to solutions not Library.

### CODE REVIEW

File: docs/fsharp/tutorials/getting-started/getting-started-command-line.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ Change directories to the *App* console project and add a reference to the `Libr
 ```bash
 dotnet add reference ../Library/Library.fsproj
 ```
-Add the `Library` project to the `FSNetCore` solution using the `dotnet sln add` command:
+Add the `Console Application` project to the `FSNetCore` solution using the `dotnet sln add` command:
```
Comment (svick) [inline docs/fsharp/tutorials/getting-started/getting-started-command-line.md]: The project is named `App`, not `Console Application`.

Comment (Vedant-Koditkar) [inline docs/fsharp/tutorials/getting-started/getting-started-command-line.md]: My bad I'll just change it?

**Comment by Vedant-Koditkar** [comment]
@svick I have changed the "Console Application" to "App".

**Review by mairaw — APPROVED** [review]
Thanks @Vedant-Koditkar for making the fix!

**Pull request merged**